README.md

```diff
@@ -0,0 +1,14 @@
+
+# #!book
+
+A FOSS hacker's guide to CLI, privacy, security, self-hosting, and the Internet
+
+## Table of Contents
+
+### [1-Internet](./1-Internet/)
+#### [1-Internet/1-intro](./1-Internet/1-intro.md)
+#### [1-Internet/2-email](./1-Internet/2-email.md)
+#### [1-Internet/3-irc](./1-Internet/3-irc.md)
+### [2-Security](./2-Security/)
+#### [2-Security/1-intro](./2-Security/1-intro.md)
+#### [2-Security/2-hashing](./2-Security/2-hashing.md)
```

Comment:
this is missing a section

Reply:
note to self to re-run `generate-readme.py`